libkbfs/folder_update_prepper.go

```diff
@@ -331,6 +331,13 @@ type pathTreeNode struct {
 	mergedPath path
 }
 
+type prepFolderFlags byte
+
+const (
+	// The blocks of indirect files should be copied.
+	prepFolderCopyIndirectFileBlocks prepFolderFlags = 1 << iota
```

Comment:
> It's more verbose, but perhaps less error-prone. What do you think?

Why less error-prone? Wouldn't someone be just as likely to mistype the wrong "behavior" as they are to forget to add a flag?